### ISSUE TITLE
Make map tips appear for GPX waypoints when moused over

### ISSUE THREAD

**Issue by wonderchook** [Cadasta/cadasta-platform#690]
Seeing the timestamp of GPX waypoints can make it easier for people to differentiate when digitizing from them.  

@dpalomino thoughts on this?

**Comment by dpalomino** [comment]
Yes, I think that would be nice to have. And maybe we can also show the waypoint name (<name>) of the waypoint together with the timestamp. The comment (<cmt>) and description (<desc>) is probably too much... What do you think?

Link to the formal description: http://www.topografix.com/GPX/1/1/#type_wptType

Real examples of collected waypoints showing this info: 

<wpt lat="6.415887" lon="3.362980">
<ele>7.657030</ele>
<time>2015-12-07T12:09:29Z</time>
<name>TMR005</name>
<cmt>TOMARO / CHIKU BOUNDARY</cmt>
<sym>Flag, Blue</sym>
</wpt>

<wpt lat="6.415950" lon="3.363082">
<ele>9.346596</ele>
<time>2015-12-07T12:17:55Z</time>
<name>209</name>
<sym>Flag, Blue</sym>
</wpt>

**Comment by dpalomino** [comment]
User Story 3.05 in backlog.

**Comment by Sanny26** [comment]
I would like to work on this issue. Can someone point me to some resource/example/docs about this issue/use case?

**Comment by wonderchook** [comment]
@Sanny26 GPX is a pretty well known standard. Read up on that. Not sure what other information I can share with you.

**Comment by clash99** [comment]
@SteadyCadence - Is this what you were mentioning today about mouseovers?
